src/styles/_catalog.scss

```diff
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  max-width: ($card-width + $gap) * 4 - $gap;
   margin: 0 auto;
 
+  &__title {
+    text-align: center;
+    margin-bottom: 50px;
+  }
```

Comment:
that won't work:
![image](https://user-images.githubusercontent.com/37161315/169843275-e4df5602-e76d-45d3-bfc8-bf880fa8e149.png)
use only grid properties to center elements
```suggestion
```